claim/README.md

```diff
@@ -585,13 +585,13 @@ If a Netdata Agent is running, the Space's administrator can connect a node usin
 additional command line parameters:
 
 ```sh
--W "claim -token=TOKEN -rooms=ROOM1,ROOM2"
+-W "claim -token TOKEN -rooms ROOM1,ROOM2"
 ```
 
 For example:
 
 ```sh
-/usr/sbin/netdata -D -W "claim -token=MYTOKEN1234567 -rooms=room1,room2"
+/usr/sbin/netdata -D -W "claim -token MYTOKEN1234567 -rooms room1,room2"
```

Comment:
both were correct

Reply:
We should keep the original here, with the = ? 

@cpipilas , @ilyam8 , @Ferroin ?

Just double checking before I push this.

Reply:
Yes, we should keep the original.

Reply:
Yes.

Reply:
Thanks, @ilyam8 . Switched back! If you can put up an approval on this, I'll push this evening.